.github/workflows/Tutor_CI.yml

```diff
@@ -13,8 +13,15 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: 3.7
+    - name: Install dependencies(flake8 and pytest included)
+      run: |
+        chmod +x ./bootstrap.sh
```

Comment:
File permissions should be tracked by Git. Changing the permissions here implies that they are not.

Reply:
Without that, the CI is failed with `./bootstrap.sh: Permission denied` error.

Reply:
Permission updated.